### DISCUSSION TITLE
Automatically changing a field

### DISCUSSION THREAD

**Discussion by Drjacky** [dropwizard/dropwizard#6442] [General]
This is my data class:
```koltin
data class SampleC(
    val somethingOkay: String,
    val isRemoteAvailable: Boolean
)
```
But when I use jackson to build the json, it change the `isRemoteAvailable` to `remoteavailable`!

- Is there any reason for this?
- Can we configure/disable this behaviour?

**Comment by joschi** [comment]
This is strictly speaking a question about Jackson and not Dropwizard. 😉 

You can customize the naming of a single class field with the [`@JsonProperty`](https://fasterxml.github.io/jackson-annotations/javadoc/2.14/com/fasterxml/jackson/annotation/JsonProperty.html) annotation.

If you're looking to change the naming strategy of all fields of the class, you can provide a custom naming strategy via [`@JsonNaming`](https://fasterxml.github.io/jackson-databind/javadoc/2.14/com/fasterxml/jackson/databind/annotation/JsonNaming.html) and an implementation of [`PropertyNamingStrategy`](https://fasterxml.github.io/jackson-databind/javadoc/2.14/com/fasterxml/jackson/databind/PropertyNamingStrategy.html).
